lms/static/scripts/ui-playground/components/EditAssignment.tsx

```diff
@@ -188,25 +188,20 @@ export default function EditAssignmentPage() {
         }
       >
         <Library.Pattern title="Edit assignment: initial view">
-          <p>
-            When transitioning from a launched assignment into the
-            edit-assignment flow, we want to take into account that the
-            assignment has already been configured.
-          </p>
-          <p>We want to make sure we:</p>
+          <p>Actions available to the user in this view:</p>
           <ul>
             <li>
-              {"Don't"} make instructors re-select content if they {"don't"}{' '}
-              want to;
+              Commit (save) assignment settings including any updated group
+              settings.
             </li>
+            <li>Exit (cancel) the edit flow.</li>
             <li>
-              Provide an obvious mechanism to exit the edit flow and get back to
-              the launched assignment
+              Elect to change the content for the assignment. This takes the
```

Comment:
```suggestion
              Change the content for the assignment. This takes the
```

"Elect to" seems redundant to me.